source/views/menus/lib/__tests__/menu-item.mock.js

```diff
@@ -1,23 +1,27 @@
 // @flow
 import type {MenuItemType} from '../../types'
 
-export const item: ({|cor_icon?: Object, station?: string, special?: boolean|}) => MenuItemType = ({'cor_icon': corIcon={}, station='', special=false}) => ({
+export const item: (
+  {|cor_icon?: Object, station?: string, special?: boolean|},
+) => MenuItemType = (
+  {cor_icon: corIcon = {}, station = '', special = false},
+) => ({
   connector: '',
-  'cor_icon': corIcon,
+  cor_icon: corIcon,
   id: '',
   description: '',
   label: '',
   monotony: {},
-  'nutrition_link': '',
+  nutrition_link: '',
   nutrition: {},
   options: [],
   rating: '',
   special: special,
   station: station,
-  'sub_station': '',
-  'sub_station_order': '',
-  'sub_station_id': '',
+  sub_station: '',
+  sub_station_order: '',
+  sub_station_id: '',
```

Comment:
These… I guess I'll add eslint-ignore comments to these spots to ignore the camelcase warnings.